sdk/identity/azure-identity/azure/identity/_internal/managed_identity_client.py

```diff
@@ -64,6 +64,11 @@ def _process_response(self, response: PipelineResponse, request_time: int) -> Ac
 
         expires_on = int(content.get("expires_on") or int(content["expires_in"]) + request_time)
         content["expires_on"] = expires_on
+        if "refresh_in" not in content:
+            refresh_in = expires_on - request_time
+            if refresh_in > 60 * 60 * 2:  # 2 hours
```

Comment:
Nit: For the sake of consistency with other languages:
```suggestion
            if refresh_in >= 60 * 60 * 2:  # 2 hours
```